src/vivarium_census_prl_synth_pop/tools/jobmon.py

```diff
@@ -0,0 +1,83 @@
+import shutil
+import uuid
+from pathlib import Path
+from typing import Union
+
+from jobmon.client.tool import Tool
+from loguru import logger
+
+
+def run_make_results_workflow(
+    raw_output_dir: Path,
+    final_output_dir: Path,
+    mark_best: bool,
+    test_run: bool,
+    artifact_path: Union[str, Path],
+    queue: str,
+    peak_memory: int,
+    max_runtime: str,
+    project: str,
+) -> None:
+    """Creates and runs a jobmon workflow to build results datasets
+    from the raw data output by observers
+    """
+    logger.info(f"Starting make_results workflow {final_output_dir}")
```

Comment:
Nit: Make this something like "saving to final_output_dir", "writing to", or :"processing results at".

Reply:
I don't mind changing, but I do want to clarify that the word "workflow" is a very jobmon-y term and I was hoping to capture that here.